app/src/main/java/deakin/gopher/guardian/view/patient/patientdata/heartrate/ui/components/HeartRateChart.kt

```diff
@@ -29,8 +35,30 @@ fun HeartRateChart(heartRates: List<HeartRate>) {
         }
 
     val chartEntryModelProducer = ChartEntryModelProducer(getEntries)
+    val heartRateChartLineSpec = arrayListOf<LineChart.LineSpec>()
```

Comment:
You should be able to use `mutableListOf<LineChart.LineSpec>` - it's a minor thing but more native Kotlin

Reply:
Hi Lachlan, I have fixed that up, no issues.